README.md

```diff
@@ -60,25 +60,9 @@ npm link @spinnaker/kayenta
 
 You should only have to run these commands once.
 
-Next, run `WATCH=true yarn lib` at the root of this repository. In a separate terminal,
+Next, run `yarn build --watch` at the root of this repository. In a separate terminal,
 run `yarn start` at the root of the main Deck repository.
```

Comment:
I kind of can't believe the readme tells developers to build the library and do all this stuff when they can (until this PR gets merged?) just run deck directly from this codebase